.github/workflows/QA.yaml

```diff
@@ -14,6 +14,7 @@ jobs:
       - name: Set up Python ${{ matrix.python }}
```

Comment:
I know I missed it in a previous review, but this is wrong, we do not have a matrix here

Reply:
Right!